src/main/java/pl/orki/hackathon/webapp/user/control/UserService.java

```diff
@@ -31,4 +51,20 @@ public User createUser(User user) {
         user.setPassword(bCryptPasswordEncoder.encode(user.getPassword()));
```

Comment:
I think this method should be private now and placed below `createUserBand` and `createUserVenue`. If you make this method private remove `@Transactional` annotation, because it won't change anything on private methods.